files/bootstrap.sh

```diff
@@ -187,6 +258,15 @@ fi
 KUBELET_CONFIG=/etc/kubernetes/kubelet/kubelet-config.json
 echo "$(jq ".clusterDNS=[\"$DNS_CLUSTER_IP\"]" $KUBELET_CONFIG)" > $KUBELET_CONFIG
 
+# set kubeReserved in the kubelet-config.json
+MEMORY_GI=$(free --si -g | grep Mem | awk '{print $2}')
```

Comment:
1. free **--si** gives you Gib instead of Gib.  It's '65' vs '62' difference on a system with 62Kib memory.
2. I'd prefer to get ride of the multiplier arg in the function. we can normalize  available_memory to Mib and available_cpu to millicpu before calling it. 
3. I'd prefer to use Mib instead Gib to calculate it preciously.
4. i prefer to use `cat /proc/meminfo  | grep MemTotal | awk '{print $2}` to denote it's memory**Total** (but use free is fine to me :D).

Reply:
Good catch on backwards --si arg.

The only reason we had the multiplier arg is because the `regions` (`start` and `end`) args were easier to read. i.e., 64 instead of 64*1024. Not a very good reason in hindsight, better to have all values in the same unit (Mi or milicpu)!